calico_containers/tests/st/libnetwork/test_libnetwork.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from tests.st.test_base import TestBase
+
+
+class LibnetworkTests(TestBase):
+
+    def test_moving_endpoints(self):
```

Comment:
Maybe `@skip` tests that haven't been written yet - easy to track, and we get stats for that too.